src/NServiceBus.Transport.RabbitMQ/RabbitMQTransport.cs

```diff
@@ -276,19 +294,50 @@ internal void ValidateAndApplyLegacyConfiguration()
                 return;
             }
 
+            VaildateTopologyFactory();
+            ValidateConnectionString();
+
+            RoutingTopology = TopologyFactory(UseDurableExchangesAndQueues);
+            ConnectionConfiguration = ConnectionConfiguration.Create(LegacyApiConnectionString);
+
+            // Uses the legacy management API connection string or build the string from the legacy broker connection configuration
+            ManagementConnectionConfiguration = !string.IsNullOrEmpty(LegacyManagementApiConnectionString)
+                ? ConnectionConfiguration.Create(LegacyManagementApiConnectionString, isManagementConnection: true)
+                : ConnectionConfiguration.Create(BuildManagementConnectionString(ConnectionConfiguration), isManagementConnection: true);
+        }
+
+        void VaildateTopologyFactory()
+        {
             if (TopologyFactory == null)
             {
                 throw new Exception("A routing topology must be configured with one of the 'EndpointConfiguration.UseTransport<RabbitMQTransport>().UseXXXXRoutingTopology()` methods. Most new projects should use the Conventional routing topology.");
             }
+        }
 
-            RoutingTopology = TopologyFactory(UseDurableExchangesAndQueues);
-
+        void ValidateConnectionString()
+        {
             if (string.IsNullOrEmpty(LegacyApiConnectionString))
             {
                 throw new Exception("A connection string must be configured with 'EndpointConfiguration.UseTransport<RabbitMQTransport>().ConnectionString()` method.");
             }
 
-            ConnectionConfiguration = ConnectionConfiguration.Create(LegacyApiConnectionString);
+            if (string.IsNullOrEmpty(LegacyManagementApiConnectionString))
+            {
+                throw new Exception("A management API connection string must be configured with 'EndpointConfiguration.UseTransport<RabbitMQTransport>().ManagementConnectionString()` method.");
+            }
```

Comment:
Even if we need to support the legacy API, the management connection string doesn't need to be provided if `DoNotUseManagementClient` is set.